### DISCUSSION TITLE
tweetBaseState file and feedState file

### DISCUSSION THREAD

**Discussion by loaishawar** [TheAlphamerc/flutter_twitter_clone#196] [Q&A] [answered]
Hi 

what this difference between tweetBaseState file and feedState file ?

i add this file to my local and i do not see any changes , can you please let me know where i can see the notice.

as will user can't see the  update functionality for own posts?

thanks

**Comment by TheAlphamerc** [comment, marked as answer]
Currently `feedState` is managing the whole app state related to tweets and this doesn't seem as per standard so I am working on improving the app architecture. That's why I created the `tweetBaseState` class. To check this new architecture have a look at the `experimental` branch.
